packages/ts/react-crud/src/autoform.tsx

```diff
@@ -325,12 +332,13 @@ export function AutoForm<M extends AbstractModel>({
 
   return (
     <div className={`auto-form ${className ?? ''}`} id={id} style={style} data-testid="auto-form">
-      <VerticalLayout className="auto-form-fields">
+      <VerticalLayout className="auto-form-fields" onKeyDown={handleKeyDown}>
         {layout}
         {formError ? <div style={{ color: 'var(--lumo-error-color)' }}>{formError}</div> : <></>}
       </VerticalLayout>
       <div className="auto-form-toolbar">
         <Button
+          id="form-submit"
```

Comment:
In general we should avoid adding any IDs to our components, as it can result in duplicate IDs if you use multiple of them. Does this have a specific purpose?